### DISCUSSION TITLE
plot_model on two variable

### DISCUSSION THREAD

**Discussion by AndIaAnd** [pycaret/pycaret#3107] [Q&A]
Please help, I need to make a plot of two columns, but the only available plot is the number (count) of and certain variable, but I really need to make a plot of two variables, preferably in this form for 'distribution'
Thank for you answer !
plot_model(kmeans, plot = 'distribution', feature = ('a':'b'))

**Comment by kilickursat** [comment]
Can you replace `plot="cluster"`, and add `label=True`?

`plot_model(kmeans, plot="cluster", feature=("a":"b"), label=True)`

**Reply by AndIaAnd** [reply]
plot_model(kmeans, plot="cluster", feature=("area":"id"), label=True)
                                                      ^
SyntaxError: invalid syntax

**Reply by kilickursat** [reply]
(https://pycaret.readthedocs.io/en/stable/api/clustering.html?highlight=plot_model#pycaret.clustering.plot_model)
